shortcut_composer/api_krita/wrappers/document.py

```diff
@@ -35,6 +37,20 @@ def active_node(self, node: Node) -> None:
         """Set active `Node`."""
         self.document.setActiveNode(node.node)
 
+    def create_node(self, name: str, node_type: NodeType) -> Node:
+        """
+        Create a Node.
```

Comment:
```suggestion
        Create a Node.
        
```
PEP257 wymaga interlinii po pierwszej linii opisu.